test/testdrive/introspection-sources.td

```diff
@@ -368,11 +368,11 @@ true true true true true true
 
 > SELECT
     records > 2 * 1000 AND records < 1.1 * 2 * 1000,
-    size > 0.5 * 0.9 * (172 + 72) * 1000,
-    size < 2 * 1.1 * (180 + 72) * 1000,
-    capacity > 0.5 * 0.9 * (172 + 72) * 1000,
-    capacity < 2 * 1.1 * (180 + 72) * 1000,
-    allocations > 1000
+    size > 0.4 * 100 * 1024,
+    size < 2.2 * 100 * 1024,
+    capacity > 0.4 * 100 * 1024,
+    capacity < 2.2 * 100 * 1024,
```

Comment:
It is odd to change the middle estimate here without changing the argument in the comment above (lines 333-342). Do we really need to change the middle estimate or just enlarge the range? If the former, would you provide a justification in the comment as well?

Also, @def- : Would you like to take this test for another spin before we merge this PR to avoid a regression of #23509?

Reply:
I'm more and more convinced that we should not try to have precise measurements here. The sizes reported here are eventually-consistent and highly depend on other parts of the code. Changes in Differential often change size, capacity, allocations slightly, and it's hard to connect the dots. For this reason, I'd like to move the tests to an order-of-magnitude approach where the values should make sense, but not precise.